### ISSUE TITLE
Modify the code for PrintVersion so it works also for RTI Micro

### ISSUE THREAD

**Issue by javmorcas** [rticommunity/rtiperftest#45]
Blocked by #78

**Comment by javmorcas** [comment]
@alfonso-soto, for the time being we are just interested in doing this for *RTI Perftest* as well as for the version of *Connext DDS Pro*, in a close future we will include the changes also for *Connext Micro*, once we have the code there.

**Comment by softwarejimenez** [comment]
Duplicate of #54

**Comment by javmorcas** [comment]
Merged into master #179
